BaseTools/Conf/build_rule.template

```diff
@@ -573,6 +582,39 @@
       "$(SLINK)" $(SLINK_FLAGS) $(OUTPUT_DIR)(+)${s_dir}(+)${s_base}.slib $(OUTPUT_DIR)(+)${s_dir}(+)${s_base}.obj
       otool -t $(OUTPUT_DIR)(+)${s_dir}(+)${s_base}.slib | hex2bin.py ${dst}
 
+#MSCHANGE

```

Comment:
This area is not part of the OemData/ProductDataFormatter changes.

Reply:
It seems to be from this 2311 commit which is strange in that `uefibot` is the author https://github.com/microsoft/mu_basecore/commit/d188c8dffb0d6af60d7fbc5ba8634b9ce3ec217a.

Reply:
Added.